helper/schema/schema.go

```diff
@@ -366,9 +378,62 @@ func (m schemaMap) Diff(
 		}
 	}
 
-	// Go through and detect all of the ComputedWhens now that we've
+	// Go through and detect/handle all of the ComputedWhens now that we've
 	// finished the diff.
-	// TODO
+	for k, schema := range m {
```

Comment:
Here be dragons. Lots of looping logic below. Would love to come up with a more elegant way to do this. Considered attempting to thread ComputedWhen knowledge/context into the original diff generation rather than this post-hoc munging, but initial spikes felt even more messy than this.